sdks/python/apache_beam/io/gcsio.py

```diff
@@ -21,13 +21,13 @@
 https://github.com/GoogleCloudPlatform/appengine-gcs-client.
 """
 
+import cStringIO as StringIO
```

Comment:
Thanks.  I don't we need to justify this in the code; for example, it is standard practice to use "cPickle" over "pickle", and we don't make such a comment when we do such imports.